### PR TITLE
Issue 16.jb.1

### PR DESCRIPTION
**Pull request by jbutcher21**
Issues #16, #11, #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,56 +70,80 @@ This will step you through the process of adding the data sources, features, att
...
+  -c CHILD_DATABASE_NAME, --child_database_name CHILD_DATABASE_NAME
+                        the name of the database file to create
+```
+
+Typical use:
+```console
+python3 openc-load-childb.py -i ./input -c ./input/child.db
+```
+
+- The -i should be the directory is where you downloaded the Open Corporates data files.
```
Comment (antaenc) [inline README.md:95]: No need for is before the where

File: README.md [hunk 2]
```diff
@@ -113,13 +152,12 @@ optional arguments:
...
+python3 openc-officers.py -i ./input/officers.csv.gz -o ./output/officers.json -t ./input/temp.db -l output/officers-log.json
 ```
 
-- The input file set should always be the openc-files.json that contains the file locations of the files you downloaded.
-- The output file directory must be a directory, not a file.  There will be one json file output named after each officer csv file
-registered in the input file set.
-- The log file is optional but gives you the statistics and examples of each attribute that was mapped.
-- You can use the -d parameter to change the data source code the mapper assigns if you would rather use something other than the default.
-
+- The -i is location of the Open Corporates officers data file.
```
Comment (antaenc) [inline README.md:158]: is the location

**Review by antaenc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jbutcher21) [reply README.md:95]: fixed

File: README.md (hunk 2 above)
Comment (jbutcher21) [reply README.md]: fixed

**Review by antaenc — APPROVED** [review]

**Pull request merged**